projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/cutsomer-ticketing.ts

```diff
@@ -0,0 +1,38 @@
+import { standardUser } from '../../sample-data/shared-users';
+import { generateMail, randomString } from '../user';
+import { clickHamburger } from '../checkout-flow';
+
+export function loginRegisteredUser() {
+  standardUser.registrationData.email = generateMail(randomString(), true);
+  cy.requireLoggedIn(standardUser);
+  cy.reload();
+}
+
+export function openMyAccountDropDown(){
+  cy.visit('/');
+  cy.wait(5000);
```

Comment:
Instead of waiting a period of time, can you waitFor an element on the screen? For instance for the My Account Menu Id

Reply:
Agree. Renamed to `clickMyAccountMenuOption`

Reply:
Done. I added the wait for a more generic parent element `cx-header` that is present in both Mobile and Desktop.